library/include/borealis/core/application.hpp

```diff
@@ -140,6 +140,12 @@ class Application
      */
     static bool loadFontFromMemory(std::string fontName, void* data, size_t size, bool freeData);
 
+    /**
+     * Wrapper for adding font fallback with nvgAddFallbackFontId.
+     * Returns true if both font stash exists and the operation succeeded.
+     */
+    static bool addFontFallback(NVGcontext *ctx, std::string baseFontName, std::string fallbackFontName);
```

Comment:
This method should not ask for a NVG context, and instead reuse Application::getNVGContext()